README.md

```diff
@@ -6,20 +6,28 @@ Created and Open Sourced thanks to [LocalBini.com](http://Localbini.com) by [pdy
 
 ![Python 3.8+](https://img.shields.io/badge/python-v3.8-blue) ![Black](https://img.shields.io/badge/code%20style-black-000000.svg)
 
-## Settings:
-`CLIENT_SECRET` For decoding authorization jwt. Defaults 'secret'.\
-`COGNITO_PUBLIC_KEYS` For validating IdToken signature when Cognito authentication enabled.\
-(simply paste from https://cognito-idp.{your aws region}.amazonaws.com/{your pool id}/.well-known/jwks.json) \
-`COGNITO_ALLOWED_CLIENTS` Allowed Cognito clients id's. String in format "id1,id2,id3". Used for verifying IdToken signature.\
-`PRINT_TRACEBACK` More verbose errors. Defaults False (set as "0" or "1").\
-`LOGGING_LEVEL` For logging level. Defaults INFO.\
-`EXPIRATION_KEY` for defining the expiration key used in authorization default "exp" according to JWT standard.
-`ALLOWED_ISS` for defining the allowed issuer of JWT - Security feature.
+## Configuration
+
+Lambdalizator can be configured using the following environment variables: 
+- `INTERNAL_AUTH_JWK` - a JSON Web Key that can be used for encoding and decoding internal JWTs.
```

Comment:
Maybe point out what internal JWT would/should be?

Reply:
I could add something, but to be honest I think this projects need a lot more documentation to explain the whole concept. Would it be ok to leave it for later? I'll have more time to write something in two weeks :)